src/Datasets/Dataset.php

```diff
@@ -290,6 +291,30 @@ public function apply(Transformer $transformer) : self
         return $this;
     }
 
+    /**
+     * Reverse a transformation to the dataset.
+     *
+     * @param \Rubix\ML\Transformers\Transformer $transformer
+     * @throws \Rubix\ML\Exceptions\RuntimeException
+     * @return static
+     */
+    public function reverse(Transformer $transformer) : self
```

Comment:
I think we can come up with a more precise name for this method. Let's keep this in mind.

Reply:
Looks good to me. I will update. What about interface name?